server/webhook_worker.go

```diff
@@ -57,6 +57,8 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 		return err
 	}
 
+	wh.CheckWatcherUser(ww.p, msg.InstanceID)
```

Comment:
`CheckIssueWatchers` seems like a better name

```suggestion
	wh.CheckIssueWatchers(ww.p, msg.InstanceID)
```